### ISSUE TITLE
Show/Hide graphic Cue Off

### ISSUE THREAD

**Issue by mawillia1** [bitfocus/companion-module-h2r-graphics#62]
I was hoping to be able to select and de-select graphics for the preview window, but H2R_Graphics: Show/Hide graphic function doesn't quite function as expected. The "Cue On" option cues up the graphic in the preview window, just as if clicking on the preview button next to the graphic (as expected), but the "Cue Off" option does does not just de-select the previewed graphic (the way a mouse click does); It also passes graphic through to the program window. It functions the same as combining the "Cue Off" and "Show" functions. Can the "Cue Off" function be isolated?

Also, Is is possible to add a toggle function for the cue on/off for the preview output (just like the show/hide toggle for the program output) so it would not require separate keys?   

Overall, this is a very useful interface and I hope it will continue to evolve!

**Comment by iamjohnbarker** [comment]
You are right - I will set up a fix for this but it has to happen in the coming v2.22 of H2R Graphics. This change will allow you to use the "Toggle Cued On/Off" in Companion to do exactly what you want. Same applies for the other "Cued on" and "Cued off" options.

Note that Companion today already has the Toggle for preview in the latest version

![Image](https://github.com/user-attachments/assets/d9af0b68-d5c7-4236-913a-5d3458f9c1df)
